project.json

```diff
@@ -1,6 +1,6 @@
 {
-  "serviceRole": "",
-  "serviceType": "",
+  "serviceRole": "exoservice-go",
+  "serviceType": "exoservice-go-type",
```

Comment:
It's used in `service.yml` and `docker-compose.yml`. I don't know if you can remember that error we got in exoservice template because `type` was not provided to exorelay. I don't know if it is also the case for `exoservice-go`